### PR TITLE
262 books buyback hardcoded endpoints

### PR DESCRIPTION
**Pull request by danielfeinblatt**
## Feature Description/Implementation
Implement hardcoded dumb endpoint for books buyback GET detail info, POST to create a new buyback, DELETE to delete a buyback, and updates.
## Dependencies
Will allow frontend to integrate as they code, rather than waiting.
## Everything Else
Need to clarify structure of the update request

### CODE REVIEW

**Review by caseyszilagyi — CHANGES_REQUESTED** [review]
One more thing: does this include the buyback list endpoint (GET /api/v1/buybacks for displaying on the buyback list page)? Or is that more difficult to hardcode?

File: backend/buybacks/views.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                    "book_title": "Moby Dick",
+                    "quantity": 4,
+                    "unit_buyback_price": 15.0
+                }],
+                "num_books": 9,
+                "num_unique_books": 2
+            },
+            status=status.HTTP_200_OK)
+
+    def update(self, request, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (danielfeinblatt) [reply backend/buybacks/views.py:84]: Okay, so you don't need anything returned? Just a confirmation it succeeded?

**Comment by danielfeinblatt** [comment]
> One more thing: does this include the buyback list endpoint (GET /api/v1/buybacks for displaying on the buyback list page)? Or is that more difficult to hardcode?

That is hardcoded under the `retrieve` method in the `RetrieveUpdateDestroyBuybackAPIView`. Let me know if this is not what you are looking for.

File: backend/buybacks/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (caseyszilagyi) [reply backend/buybacks/views.py:86]: Right now, anytime we hit a modify/create/delete endpoint, we just check for a 200/201/etc status to ensure it's successful and don't do anything with the response. So doesn't really matter what the response is. Just thought I'd let you know, but it's fine like this

**Review by caseyszilagyi — APPROVED** [review]

**Pull request merged**